released/packages/coq-rupicola/coq-rupicola.0.0.1/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+authors: [
+  "Massachusetts Institute of Technology"
+]
+maintainer: "Jason Gross <jgross@mit.edu>"
```

Comment:
What is the meaning of  maintainers in OPAM?

Reply:
I think it's "maintainer of the opam package", but someone please correct me if I'm wrong.

Reply:
That is right.